spec.html

```diff
@@ -5546,6 +5794,125 @@ <h1>CopyDataProperties ( _target_, _source_, _excludedItems_ )</h1>
         <p>The target passed in here is always a newly created object which is not directly accessible in case of an error being thrown.</p>
       </emu-note>
     </emu-clause>
+
+    <emu-clause id="sec-newprivatename" aoid="NewPrivateName">
+      <h1>NewPrivateName ( _description_ )</h1>
+      <emu-alg>
+        1. Assert: Type(_description_) is String.
+        1. Return a new unique Private Name whose [[Description]] value is _description_.
```

Comment:
I'm still bothered by the term "unique" here. All records have identity and are therefore unique.

Reply:
I dropped the word "unique" here (in https://github.com/tc39/ecma262/pull/1668/commits/f429e6d7c9e9992fad633022fa775c31879d22c6), but left it in the description of Private Names (in https://github.com/tc39/ecma262/pull/1668/commits/214d6eae96bfa37b9eafc1a4f99241b6555ddaa7).